tutorials/preparing-the-vm.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Preparing the Virtual Machine
+---
+
+Complete the following steps to run Kite example code on a Cloudera Quickstart VM.
+
+* Install a VirtualBox or VMWare [Cloudera QuickStart VM][getvm] version 5.2 or later.
```

Comment:
Should note that VirtualBox is recommended. People generally have more problems with VMWare.